lib/r10k/api.rb

```diff
@@ -97,6 +97,20 @@ def parse_deployed_env(path, moduledir="modules")
     # @param path [String] Path on disk to search for Puppet environments.
     # @return [Array<Hash>] An array of hashmaps, each representing the actual state of a single environment found in environmentdir.
     def parse_environmentdir(path)
```

Comment:
We should probably add the ability to pass a custom moduledir down through as well. (That's an argument that got added after this was mocked out.)